test/clt-tests/core/test-unfreeze-defrosts.rec

```diff
@@ -0,0 +1,74 @@
+––– block: ../base/start-searchd –––
+––– input –––
+set -b +m
+––– output –––
+––– input –––
+mysql -h0 -P9306 -e "drop table if exists t; create table t(f text, s string) rt_mem_limit='128k'; insert into t values(0,'abc','string'); freeze t;" > /dev/null 2>&1
+––– output –––
+––– input –––
+sleep 5; mysql -h0 -P9306 -e "SHOW THREADS\G;"
+––– output –––
+*************************** 1. row ***************************
+TID: %{NUMBER}
+Name: work_%{NUMBER}
+Proto: mysql
+State: query
+Connection from: 127.0.0.1:%{NUMBER}
+ConnID: %{NUMBER}
+This/prev job time: %{NUMBER}#!/(s|ms|us)/!#
+Jobs done: %{NUMBER}
+Thread status: working
+Info: SHOW THREADS
+––– input –––
+for n in `seq 1 10000`; do mysql -h0 -P9306 -e "insert into t values(0, 'abc', 'string string string stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring stringstring string');"; echo -n .; done 1>/dev/null &
+––– output –––
+[%{NUMBER}] %{NUMBER}
+––– input –––
+sleep 7; mysql -h0 -P9306 -e "select count(*) from t\G; unfreeze t;"
+––– output –––
+*************************** 1. row ***************************
+count(*): %{NUMBER}
```

Comment:
It's not enough that it's a random number. I'm afraid with a random number the test makes no sense.

Reply:
I suppose it is possible to make dependencies between several values of the number of rows. In one case they will change, in another case they will be unchanged. exact values cannot be set, as they will be different in each test run and in CI the speed is faster than the local machine.

Reply:
Fixed